src/main/scala/treadle/executable/SymbolTable.scala

```diff
@@ -382,7 +383,7 @@ object SymbolTable extends LazyLogging {
           moduleMemoryToMemorySymbol(moduleMemory) += memorySymbols.head
 
 
-        case stop @ Stop(info, args, clockExpression, enableExpression)   =>
+        case stop @ Stop(info, _, clockExpression, enableExpression)   =>
```

Comment:
I'm not sure I understand what this has to do with printf

Reply:
Nothing, it was just an unused instantiation. Makes file perfectly clean.
I can undo if you'd like

Reply:
If it's going to be combined with another PR, I would think it would be better to combine it with #114.